msautotest/misc/tiles_scales_z2.map

```diff
@@ -0,0 +1,43 @@
+#
+# Test for mode=tile and the use of MAXSCALE and MINSCALE when UNITS=dd.
+# This test should generate a blank tile because the layer is out of scale
+# (zoom level too high).
+#
+# REQUIRES: INPUT=OGR OUTPUT=PNG SUPPORTS=PROJ
+#
+# RUN_PARMS: tile_scale_empty.png [MAPSERV] QUERY_STRING="map=[MAPFILE]&MODE=tile&TILEMODE=gmap&TILE=1+1+2&LAYERS=canada-poly" > [RESULT_DEMIME]
+#
+MAP
+  NAME "TILE_SCALE_TEST"
+  IMAGETYPE PNG
+  EXTENT -140.992892 41.976786 -55.630945 71.990315
+  SIZE 400 400
+  DEBUG 5
+  UNITS dd
+  
+  PROJECTION
+    "init=epsg:4326"
+  END #projection
+
+  LAYER
+    DEBUG 5
+    NAME "canada-poly"
+    TYPE POLYGON
+    STATUS DEFAULT
+    CONNECTIONTYPE OGR
+    CONNECTION "data/canada.dgn"
+    DATA "elements"
+    MINSCALE 30000000
```

Comment:
`MINSCALE` and `MAXSCALE` are deprecated. These should be replaced with `MINSCALEDENOM` and `MAXSCALEDENOM`.